scripts/test_ping.py

```
@@ -0,0 +1,30 @@
+
+    start = time.time()
+    response = b''
+    while time.time() - start < 10:
+        try:
+            data = ser.read(1024)
+            if data:
+                response += data
+                print(f'Received: {data.decode("utf-8", errors="replace")}')
+                if b'{' in response and b'}' in response:
+                    break
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)